### ISSUE TITLE
Droplet instance not running

### ISSUE THREAD

**Issue by dakshshah96** [InstaPy/InstaPy#212]
I'm an absolute beginner to this and have followed the Ubuntu instructions and set up my droplet with Digital Ocean according to the docs. I get the following error when I run `python3.6 ./quickstart.py`

```
root@instapy:~/InstaPy# python3.6 ./quickstart.py
Traceback (most recent call last):
  File "./quickstart.py", line 2, in <module>
    from instapy import InstaPy
  File "/root/InstaPy/instapy/__init__.py", line 1, in <module>
    from .instapy import InstaPy
  File "/root/InstaPy/instapy/instapy.py", line 5, in <module>
    from pyvirtualdisplay import Display
ModuleNotFoundError: No module named 'pyvirtualdisplay'
Error in sys.excepthook:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/apport_python_hook.py", line 63, in apport_excepthook
    from apport.fileutils import likely_packaged, get_recent_crashes
  File "/usr/lib/python3/dist-packages/apport/__init__.py", line 5, in <module>
    from apport.report import Report
  File "/usr/lib/python3/dist-packages/apport/report.py", line 30, in <module>
    import apport.fileutils
  File "/usr/lib/python3/dist-packages/apport/fileutils.py", line 23, in <module>
    from apport.packaging_impl import impl as packaging
  File "/usr/lib/python3/dist-packages/apport/packaging_impl.py", line 23, in <module>
    import apt
  File "/usr/lib/python3/dist-packages/apt/__init__.py", line 23, in <module>
    import apt_pkg
ModuleNotFoundError: No module named 'apt_pkg'

Original exception was:
Traceback (most recent call last):
  File "./quickstart.py", line 2, in <module>
    from instapy import InstaPy
  File "/root/InstaPy/instapy/__init__.py", line 1, in <module>
    from .instapy import InstaPy
  File "/root/InstaPy/instapy/instapy.py", line 5, in <module>
    from pyvirtualdisplay import Display
ModuleNotFoundError: No module named 'pyvirtualdisplay'
root@instapy:~/InstaPy#
```

Anyone has any ideas what the problem might be?

**Comment by timgrossmann** [comment]
@dakshshah96 Please make sure to always post the solution to your problem, because other people will have the same problems and can then find an answer

**Comment by dakshshah96** [comment]
@timgrossmann Sure! So everyone who has this problem, here's what I found out about the errors I faced.

I was getting an error which said that the module `pyvirtualdisplay` was not present even after I had installed it. This was because I was running `python3.6 ./quickstart.py` whereas the module was present in my default python directory which was `python3.5.1`. `python3.5.1 ./quickstart.py` worked perfectly fine with no errors.
